docs/content/https/tls.md

```diff
@@ -159,6 +159,75 @@ data:
 
 If no default certificate is provided, Traefik generates and uses a self-signed certificate.
 
+### ACME Default Certificate
+
+You can configure Traefik to use an ACME provider (like Let's Encrypt) to generate the default certificate.
+The configuration to resolve the default certificate should be defined in a TLS store:
+
+!!! important "Precedence with the `defaultCertificate` option"
+
+    The `defaultCertificate` definition takes precedence on the ACME default certificate configuration.
+
+```yaml tab="File (YAML)"
+# Dynamic configuration
+
+tls:
+  stores:
+    default:
+      defaultCertResolver: myresolver
+      defaultCertDomain:
+        main: foobar
+        sans:
+          - foobar
+          - foobar
```

Comment:
do we still plan on supporting [multiple certificate stores](https://github.com/traefik/traefik/issues/4756)? ;)
If we don't, in my opinion it does not make sense to have `default` prepended on the config keys as `default` is already the name of the single store available

Reply:
This is a good point, thus I think that the removal of the multiple TLS stores definition in the configuration should be handled in another PR, if at some point we want to remove it.